src/theme/spacings.js

```diff
@@ -0,0 +1,22 @@
+import { forEach } from 'lodash';
+
+export const spacing = {
+  none: '0px',
```

Comment:
Do we really need this? and if so would it be better as just 0 seeing as unit is irrelevant and often times you want to explicitly have no border with `border: 0` as opposed to a border of `0px` width. Semantics ¯\\_(ツ)_/¯

Reply:
> Do we really need this?

I put it there just in case, so that if we ever decided to enforce the linting rules, we don't have come back to this just to add the zero :)

> ...would it be better as just 0 seeing as unit...

Agree, will make the change.